src/CNGLOB.H

```diff
@@ -84,17 +84,26 @@ typedef long long LLI;		// signed 64 bit integer
 typedef unsigned long long ULLI;
 //typedef int BOOL;	if needed: // 16 or 32 bit Boolean, matches windows.h.
 
-#undef ELIMINATE_WINDOWS		// #define to eliminate Windows dependencies
-								//   work in progress, 10-2017
-#if defined( ELIMINATE_WINDOWS)
-// typedefs based on windows.h
+/*---------------------------- Eliminate Windows ---------------------------*/
+#define	ELIMINATE_WINDOWS			// #define to eliminate Windows dependencies
```

Comment:
Do we want to keep `ELIMINATE_WINDOWS` for this PR?

Reply:
No. Let's get this PR to run without it (using `_WIN32` as necessary).